kakaemma/My-Diary-React
# bug fixes for modules index.css index.js not found

#### What does this PR do?
This PR fixes a bug of module index.css not found

#### How should this be manually tested?
Clone the application with `git clone `
install application dependencies with `npm install`
Run the application with npm start
On success, a landing page with a login form should will be displayed.

What are the relevant pivotal tracker stories?
[#162244267](https://www.pivotaltracker.com/story/show/162244267)

#### Any background context you want to add?
N/A
